doc/nrf/protocols/lte/psm.rst

```diff
@@ -0,0 +1,267 @@
+.. _cellular_psm:
+
+Power saving techniques
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Cellular technology is an example of an LPWAN (Low Power Wide Area Network) because to its coverage, high mobility, roaming capabilities, and infrastructure.
```

Comment:
This should also be "LTE" instead of "cellular".

Reply:
I suggest avoiding the term "LTE" as it confines us to 4G terminology, no?
We should also emphasize that NB-IoT and LTE-M are integral to the evolving 5G technology landscape.

Reply:
LTE-M and NB-IoT are fine.